components/DualMessageOverlay.tsx

```diff
@@ -1,21 +1,29 @@
 import React from 'react';
-import { useTranslation } from 'react-i18next';
-import { Dimensions, View } from 'react-native';
-import { Overlay } from 'react-native-elements';
-import { Column, Row, Text } from './ui';
-import { Button } from './ui/Button';
-import { Theme } from './ui/styleUtils';
+import {useTranslation} from 'react-i18next';
+import {Dimensions, StyleSheet} from 'react-native';
+import {Overlay} from 'react-native-elements';
+import {Column, Row, Text} from './ui';
+import {Button} from './ui/Button';
+import {Theme} from './ui/styleUtils';
 
 /**
  * DualMessageOverlay is like MessageOverlay but with two buttons
  *
  * NOTE: This has been added for surfacing bugs and needs to be refactored
  * before use.
  */
-export const DualMessageOverlay: React.FC<DualMessageOverlayProps> = (
-  props
-) => {
-  const { t } = useTranslation('common');
+export const DualMessageOverlay: React.FC<DualMessageOverlayProps> = props => {
+  const {t} = useTranslation('common');
+  const style = StyleSheet.create({
+    customHeight: {
+      height: props.customHeight
+        ? props.customHeight
+        : props.progress
+        ? 100
```

Comment:
Why can't we have it from theme as it already exists `Theme.MessageOverlayStyles.popupOverLay`
And why we want to create this as style instead of just height?

Reply:
Because we want to conditionally set the height property as per the set properties, hence I had to move the height values from the theme stylesheets to here.